### ISSUE TITLE
cmd/gofmt: allow same level indent for block comments

### ISSUE THREAD

**Issue by dsnet** [golang/go#38204]
Given the following code:
```go
/*
someDisabledCode()
*/
```

It will be reformatted as:
```go
/*
	someDisabledCode()
*/
```
(notice the indent it injects)

This is frustrating for users who need to maintain patches since it means that there is now a potentially massive diff when trying to comment large blocks of code in a patch.

Would it be possible for gofmt to stop trying to indent everything within the block comments?

**Comment by griesemer** [comment]
@dsnet Yes, it would be possible :-)

It was probably a mistake to try to "adjust" comments in any way with gofmt, but it also has been like this for > 10 years. We could try to just leave comments alone. It shouldn't have an effect on existing code since it would simply remain untouched.

\cc @rsc for another opinion.

**Comment by dsnet** [comment]
> but it also has been like this for > 10 years

Yep, and every year, I've probably been annoyed by this issue. It took till now for me to file an issue about it ;)

**Comment by mvdan** [comment]
This gets a thumbs up from me too. Though I have to admit, I only ever use `/*` comments to comment out chunks of code without affecting the diff too much (modulo this issue).

**Comment by hrissan** [comment]
This also frustrates me, because additional indent destroys "blame" for all commented lines. 

Looking forward for the fix)
